modules/web/src/app/wizard/step/provider-settings/provider/extended/azure/component.ts

```diff
@@ -255,6 +266,10 @@ export class AzureProviderExtendedComponent extends BaseFormValidator implements
     return internalFormValue ? internalFormValue[AutocompleteControls.Main] : '';
   }
 
+  clearLoadBalancerSKU(): void {
+    this.selectedLoadBalancerSKU = '';
```

Comment:
do we use `set selectedLoadBalancerSKU` in somewhere else ?
```suggestion
    this.form.get(Controls.LoadBalancerSKU).reset();
```